flowzone.yml

```diff
@@ -44,6 +45,42 @@
       submodules: "recursive"
       token: ${{ secrets.FLOWZONE_TOKEN }}
 
+  # FIXME: https://github.com/balena-io/deploy-to-balena-action/issues/194
+  - &updateBalenaReleaseNotes
+    name: Update balena release notes
+    run: |
+      set -ea
+      # prevent git from existing with 141
+      set +o pipefail
+
+      previous_tag="$(git --no-pager tag --list --sort=-version:refname "v*.*.*" --merged | head -n2 | tail -n1)"
+      release_notes="$(git log ${previous_tag}..${{ github.event.pull_request.head.sha || github.event.head_commit.id }} --pretty=reference | jq -R -s '.')"
+
+      app_id="$(curl "https://api.${{ inputs.balena_environment }}/v6/application?\$filter=slug%20eq%20%27${{ matrix.slug }}%27&\$select=id" \
+        -H 'Content-Type: application/json' \
+        -H 'Authorization: Bearer ${{ secrets.BALENA_API_KEY || secrets.BALENA_API_KEY_PUSH }}' \
+        | jq -r '.d[].id')"
+
+      release_id='${{ steps.balena_deploy.outputs.release_id }}'
+      # FIXME: https://github.com/balena-io/deploy-to-balena-action/issues/193
```

Comment:
We should see if someone is willing to fix this, myself included, before committing this workaround IMO.

Maybe it's a 1min fix but no one has looked yet. I'll add it to my list.

Reply:
See https://github.com/balena-io/deploy-to-balena-action/pull/246